docs/siteConfig.js

```diff
@@ -0,0 +1,81 @@
+// Documentation for these options can be found at:
+// https://docusaurus.io/docs/en/site-config
+
+const siteConfig = {
+  title: 'OpenRefine', // Title for your website.
+  tagline: 'A power tool for working with messy data.',
+  url: 'https://openrefine.github.io/',
+  baseUrl: '/docs-prototype/',
+
+  projectName: 'OpenRefine',
+  organizationName: 'OpenRefine',
+
+  // For no header links in the top nav bar -> headerLinks: [],
+  headerLinks: [
+    {doc: 'index', label: 'Home'},
+  ],
+
+  users: [],
+  customDocsPath: 'docs/source',
+  docsUrl: '',
+
+  /* path to images for header/footer */
+  headerIcon: 'img/openrefine_logo.png',
+  footerIcon: 'img/openrefine_logo.png',
+  favicon: 'img/openrefine_logo.png',
+
+  /* Colors for website */
+  colors: {
+    primaryColor: '#196581',
```

Comment:
I'd would change the primary color to be one of the 2 within the inner facets of our new logo.
![Annotation 2020-04-03 103147](https://user-images.githubusercontent.com/986438/78378720-00abec00-7597-11ea-998d-90f68f2826e2.png)

#00a0ff  <-- a bit darker
#00c8ff  <-- a bit lighter

Reply:
Hmmm, actually, even your dark one seems too agressive to me:
![image](https://user-images.githubusercontent.com/309908/78381279-6dde7200-75d5-11ea-921b-506742133691.png)

Reply:
yeah I saw that also...and why I updated the comment to say "or even a darker shade...".  just turn it down.  Just increase the K value (black).  Perhaps `#0080cc` or even darker...whatever begins to make it look nicely contrasted.